api/README.md

```diff
@@ -2,24 +2,19 @@
 
 The lightweight API is a stripped-down alternative to the WordPress REST API. Prompts will often be present on every page of a site, which can lead to performance issues as a result of a large number of uncacheable API requests.
 
-The feature requires a custom config file named `newspack-popups-config.php` at the root of the site, which will be created at plugin initialisation.
+## Configuration
 
-To include information about database, cache and time in the response, add this constant to that config file:
+The feature requires a custom config file named `newspack-popups-config.php` at the root of the site. See the main README for more information.
 
-```
-define( 'NEWSPACK_POPUPS_DEBUG', true );
-```
+## API Overview
 
-## Manually creating the config file
+| Endpoint | Frequency | Purpose |
+| :------------- | :------------- | :------------- |
+| `GET /api/campaigns/index.php` | Page load | Tells the page which prompts should be displayed. This is the [authorization endpoint](https://amp.dev/documentation/components/amp-access/#authorization-endpoint) of the `amp-access` AMP component. |
+| `GET /api/segmentation/index.php` | Page load | An analytics [config rewriter](https://amp.dev/documentation/components/amp-analytics/?format=websites#dynamically-rewrite-a-configuration), which inserts segmentation-derived custom dimensions. This feature has to be additionaly configured in Newspack Plugin's Analytics Wizard. |
```

Comment:
```suggestion
| `GET /api/segmentation/index.php` | Page load | An analytics [config rewriter](https://amp.dev/documentation/components/amp-analytics/?format=websites#dynamically-rewrite-a-configuration), which inserts segmentation-derived custom dimensions. This feature has to be additionally configured in Newspack Plugin's Analytics Wizard. |
```

Reply:
Maybe include a link to the [public documentation for custom dimensions](https://newspack.pub/support/analytics/#a-custom-dimensions) here?